admin/code/CampaignAdmin.php

```diff
@@ -416,6 +424,44 @@ public function deleteCampaign(SS_HTTPRequest $request) {
 	}
 
 	/**
+	 * REST endpoint to publish a {@link ChangeSet} and all of its items.
+	 *
+	 * @param SS_HTTPRequest $request
+	 *
+	 * @return SS_HTTPResponse
+	 */
+	public function publishCampaign(SS_HTTPRequest $request) {
```

Comment:
Done now, for publish. Delete has the same issue (existing feature, not part of this PR) - created an issue for it at https://github.com/silverstripe/silverstripe-framework/issues/5328